tests/adapters_t/batchsystems_t/test_htcondor.py

```diff
@@ -136,15 +136,19 @@ def test_get_machine_status(self):
                          MachineStatus.Available)
         self.mock_async_run_command.reset_mock()
 
-        self.mock_async_run_command.side_effect = AsyncRunCommandFailure(message="Test", error_code=123,
-                                                                         error_message="Test")
+        self.mock_async_run_command.side_effect = CommandExecutionFailure(message="Test", exit_code=123,
+                                                                          stderr="Test")
         with self.assertLogs(level='ERROR'):
-            attributes = dict(Machine='Machine', State='State', Activity='Activity', TardisDroneUuid='TardisDroneUuid')
-            # Escape htcondor expressions and add them to attributes
-            attributes.update({key: quote(value) for key, value in self.config.BatchSystem.ratios.items()})
-
-            run_async(partial(htcondor_status_updater, self.config.BatchSystem.options, attributes))
-            self.mock_async_run_command.assert_called_with(self.command)
+            with self.assertRaises(CommandExecutionFailure):
+                attributes = dict(Machine='Machine', State='State', Activity='Activity',
```

Comment:
This could be rewritten as a ``{}`` literal. Any objections?

Reply:
Nope!

Reply:
Added a new commit changing this! Could you approve again, please?